docs/sources/datasources/prometheus.md

```diff
@@ -46,8 +46,8 @@ Open a graph in edit mode by clicking the title > Edit (or by pressing `e` key w
 | `Query expression`  | Prometheus query expression, check out the [Prometheus documentation](http://prometheus.io/docs/querying/basics/).                                                                                                                                                                                                                                                                                                                                                                                        |
 | `Legend format`     | Controls the name of the time series, using name or pattern. For example `{{hostname}}` is replaced with the label value for the label `hostname`.                                                                                                                                                                                                                                                                                                                                                        |
-| `Min step`          | An additional lower limit for the [`step` parameter of Prometheus range queries](https://prometheus.io/docs/prometheus/latest/querying/api/#range-queries) and for the `$__interval` and `$__rate_interval` variables. The limit is absolute, it cannot modified by the _Resolution_ setting.                                                                                                                                                                                                             |
-| `Resolution`        | `1/1` sets both the `$__interval` variable and the [`step` parameter of Prometheus range queries](https://prometheus.io/docs/prometheus/latest/querying/api/#range-queries) such that each pixel corresponds to one data point. For better performance, lower resolutions can be picked. `1/2` only retrieves a data point for every other pixel, and `1/10` retrieves one data point per 10 pixels. Note that both _Min time interval_ and _Min step_ limit the final value of `$__interval` and `step`. |
+| `Step`              | Optionally, set the lower or upper bounds on the interval between data points, for example, set "minimum 1h" to hiint that measurements were not taken more frequently. `$__interval` and `$__rate_interval` are supported.                                                                                                                                                                                             |
```

Comment:
fyi @achatterjee-grafana.

We are not mentioning here that users can choose step mode - min/max/exact and what does each mean.

Reply:
What about something like this?:
**Use 'Minimum' or 'Maximum' step mode to set the lower or upper bounds respectively on the interval between data points. For example, set "minimum 1h" to hint that measurements were not taken more frequently. Use the 'Exact' step mode to set an exact interval between data points. `$__interval` and `$__rate_interval` are supported.**

Reply:
Perfect! 😄

Reply:
What are your thoughts @achatterjee-grafana?